docs/maps/maps-getting-started.asciidoc

```diff
@@ -76,8 +77,8 @@ the {es} index `kibana_sample_data_logs` on the shared key iso2 = geo.src.
 
 ===== Set the vector style
 
-The final step is to set the vector fill color to shade 
-the countries by web log traffic.
+All of the world countries are still a single color because the layer is using <<maps-vector-style-static, static styling>>.
+Lets use <<maps-vector-style-data-driven, data driven styling>> to shade the world countries based on which country is sending the most requests.
```

Comment:
Suggest changing this because we haven't really used "we" in this tutorial.

To shade the word countries based on which country is sending the most requests, you'll need to use <<maps-vector-style-data-driven, data driven styling>>.